### ISSUE TITLE
[Bug] zos_operator can't run the JES2 command

### ISSUE THREAD

**Issue by hyzhang6639** [ansible-collections/ibm_zos_core#645]
### Bug description

The task is as the following:
  tasks: 
    - name: Purging the job
      zos_operator:
        cmd: "\\$PJ(*)"
        verbose: false
      register:
        result

The below error is shown in z/OS operation log. It appears that a space is added before the JES2 command.
 ¥PJ(*)                                                               
IEE305I  ¥PJ(*)  COMMAND INVALID                                      
IEA631I  OPERATOR IBMUSER¥ NOW INACTIVE, SYSTEM=TVT4124 , LU=091EF1C9 
IEA630I  OPERATOR IBMUSER¥ NOW ACTIVE,   SYSTEM=TVT4124 , LU=091EF1C9 

### Ansible version

```shell
ansible [core 2.14.2]
  config file = /root/z_ansible/tvt4124/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.9/site-packages/ansible
  ansible collection location = /root/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/local/bin/ansible
  python version = 3.9.2 (default, May  8 2021, 07:45:39) [GCC 4.8.5 20150623 (Red Hat 4.8.5-44)] (/usr/local/bin/python3)
  jinja version = 3.0.3
  libyaml = True
```

### IBM z/OS Ansible core Version

v1.4.0

### IBM ZOAU version

v1.1.1

### Ansible module

zos_operator

**Comment by ddimatos** [comment]
Hello @hyzhang6639 , thank you for opening an issue. I am able to recreate this in 1.4.0,  this is regression for1.4.0, it is fixed in 1.5.0-beta.1 coming out next week. Escaping your backslash below is required for all versions. We will triage to see how we want to approach this since its fixed in next. 

```
    - name: Run operator command
      ibm.ibm_zos_core.zos_operator:
        cmd: "\\$PJ(*)"
        verbose: false
      register: result_zos_operator
```

1.5.0-beta. 1 output:
```
PLAY [zvm] ****************************************************************************************************************************************

TASK [Run operator command] ***********************************************************************************************************************
changed: [zvm]

TASK [Response run operator command] **************************************************************************************************************
ok: [zvm] => {
    "msg": {
        "changed": true,
        "cmd": "\\$PJ(*)",
        "content": [
            "EC33017A  2023047  18:44:22.25             ISF031I CONSOLE OMVSADM ACTIVATED",
            "EC33017A  2023047  18:44:22.25            -$PJ(*)",
            "EC33017A  2023047  18:44:22.43  JOB00026   $HASP890 JOB(DEFNVSAM)",
            "                                           $HASP890 JOB(DEFNVSAM)  STATUS=(AWAITING PURGE),CLASS=K,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.48  JOB00052   $HASP890 JOB(LOCALDSN)",
            "                                           $HASP890 JOB(LOCALDSN)  STATUS=(AWAITING PURGE),CLASS=K,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.53  JOB00079   $HASP890 JOB(SSHSETUP)",
            "                                           $HASP890 JOB(SSHSETUP)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.53  JOB00080   $HASP890 JOB(SSHSTRT)",
            "                                           $HASP890 JOB(SSHSTRT)   STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.53  JOB00243   $HASP890 JOB(COMPLINK)",
            "                                           $HASP890 JOB(COMPLINK)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.53  JOB00244   $HASP890 JOB(HELLOPGM)",
            "                                           $HASP890 JOB(HELLOPGM)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.54  JOB00352   $HASP890 JOB(HELLOPGE)",
            "                                           $HASP890 JOB(HELLOPGE)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.54  JOB00353   $HASP890 JOB(HELLOPGE)",
            "                                           $HASP890 JOB(HELLOPGE)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.54  JOB00354   $HASP890 JOB(HELLOPGF)",
            "                                           $HASP890 JOB(HELLOPGF)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.54  JOB00355   $HASP890 JOB(HELLOPGG)",
            "                                           $HASP890 JOB(HELLOPGG)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.54  JOB00357   $HASP890 JOB(HELLOPGH)",
            "                                           $HASP890 JOB(HELLOPGH)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.54  JOB00359   $HASP890 JOB(HELLOPGA)",
            "                                           $HASP890 JOB(HELLOPGA)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.54  JOB00360   $HASP890 JOB(HELLOPGC)",
            "                                           $HASP890 JOB(HELLOPGC)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.54  JOB00361   $HASP890 JOB(HELLOPGC)",
            "                                           $HASP890 JOB(HELLOPGC)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.55  JOB00371   $HASP890 JOB(HELLOPGI)",
            "                                           $HASP890 JOB(HELLOPGI)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.55  JOB00373   $HASP890 JOB(HELLOPGI)",
            "                                           $HASP890 JOB(HELLOPGI)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.55  JOB00374   $HASP890 JOB(HELLOPGI)",
            "                                           $HASP890 JOB(HELLOPGI)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.55  JOB00375   $HASP890 JOB(HELLOPGJ)",
            "                                           $HASP890 JOB(HELLOPGJ)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.55  JOB00377   $HASP890 JOB(HELLOPGK)",
            "                                           $HASP890 JOB(HELLOPGK)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.55  JOB00378   $HASP890 JOB(HELLOPGI)",
            "                                           $HASP890 JOB(HELLOPGI)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.58  JOB00379   $HASP890 JOB(HELLOPGI)",
            "                                           $HASP890 JOB(HELLOPGI)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.58  JOB00381   $HASP890 JOB(HELLOPGL)",
            "                                           $HASP890 JOB(HELLOPGL)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.59  JOB00382   $HASP890 JOB(HELLOPGL)",
            "                                           $HASP890 JOB(HELLOPGL)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.59  JOB00386   $HASP890 JOB(HELLOPGN)",
            "                                           $HASP890 JOB(HELLOPGN)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.59  JOB00387   $HASP890 JOB(HELLOPGO)",
            "                                           $HASP890 JOB(HELLOPGO)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.59  JOB00388   $HASP890 JOB(HELLOPGN)",
            "                                           $HASP890 JOB(HELLOPGN)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES",
            "EC33017A  2023047  18:44:22.59  JOB00389   $HASP890 JOB(HELLOPGO)",
            "                                           $HASP890 JOB(HELLOPGO)  STATUS=(AWAITING PURGE),CLASS=A,",
            "                                           $HASP890                PRIORITY=1,SYSAFF=(ANY),HOLD=(NONE),",
            "                                           $HASP890                PURGE=YES"
        ],
        "elapsed": 1.8,
        "failed": false,
        "rc": 0,
        "wait_time_s": 1
    }
}

PLAY RECAP ****************************************************************************************************************************************
zvm                        : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
```

**Comment by ddimatos** [comment]
@hyzhang6639 are you able to use IBM z/OS core versions 1.5.0-beta.1 or the coming 1.5.0 coming middle of April 2023 where this issue is resolved?

**Comment by ddimatos** [comment]
Doc issue resolved by issue #612
